### ISSUE TITLE
Block mover controls hidden on full-width block

### ISSUE THREAD

**Issue by dianeco** [WordPress/gutenberg#14817]
**Describe the bug**
The block mover controls are hidden for all full-width blocks making impossible to move them.
This regression appears at least since Gutenberg 5.3 and therefore is in WordPress 5.2 Beta 1.

**Comment by m-e-h** [comment]
Looks like the HTML changed at some point?
In addition to the full-width movers being hidden behind the admin-sidebar, the align-wide movers are now vertical and on the side instead of horizontal above the block like they once were.

The CSS selector for full width is:
```
.editor-block-list__layout .editor-block-list__block[data-align="full"] > .editor-block-mover
```
I haven't check but I'm guessing it's still there.

It's missing the target now because `.editor-block-mover` isn't a direct descendent of `[data-align="full"]` or `[data-align="wide"]` anymore. It's wrapped in `block-editor-block-list__block-edit` now.

If the current HTML is correct than the CSS selector needs changed to something like this:
```
.editor-block-list__layout .editor-block-list__block[data-align="full"] > .block-editor-block-list__block-edit > .block-editor-block-mover
```
Is this HTML change legit? Should the CSS be adjusted? @kjellr @youknowriad

**Comment by aduth** [comment]
Appears related: #12758

**Comment by aduth** [comment]
Fix pull request: #15022
